src/posts/paper/13-q-and-a.ts

```diff
@@ -230,22 +232,43 @@ You may additionally increase your budget based on how the vaccine reduces your
 Some ways you may want to change your budget in after you and your close contacts are vaccinated:
 * If you want to keep the same overall budget but live with housemates, you can increase your personal budget while maintaining the same annual risk; each housemate's activites has a reduced impact on others' microCOVID budget. I.e. if living with N housemates, before vaccination each person's allocation of microCOVIDs would have been (200 / (1 + 0.3N)). After everyone is vaccinated, this increases to (200 / (1 + 0.03N)).
 * You may have been choosing your microCOVID budget primarily based on mitigating risk to others, rather than concern for your own health; When both you and the people you are trying to protect are vaccinated, you may increase your budget by 1 / vaccine multiplier (i.e. 10x for the mRNA vaccines) while still having the same impact on them. 
-* If you are primarily concerned about hospitalization/death but unconcerned about effects of mild COVID, you may double your budget post vaccination.
+* If you are only concerned about the risk of your own hospitalization/death and unconcerned about effects of mild COVID, you may 5x your budget post vaccination.
```

Comment:
"and unconcerned about effects of mild COVID" -> and unconcerned about effects of mild COVID or potential long-term symptoms

?